Source/EasyNetQ/QueueAttribute.cs

```diff
@@ -2,19 +2,13 @@
 
 namespace EasyNetQ;
 
-[AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = false)]
```

Comment:
I suggest dropping the positional QueueName, renaming QueueName & QueueType to Name & Type, extracting ExchangeAttribute to specify exchange Name.

Reply:
Specifying ExchangeName in QueueAttribute looks a bit weird.

Reply:
Personally I'm fine with these changes but I have to note that they are breaking.